src/components/form-elements/_select.scss

```diff
@@ -1,40 +1,43 @@
-$selectHeight: rhythm(3 / 2);
-$selectBackground: $white;
+$selectHeight: 40px;
```

Comment:
I'd like to see where these sizes come from

Reply:
this is the same size, as we have in Buttons, I can move it to config if you like.

Reply:
I mean, even comment would be enough, that base size of our components is 40px and that comes from designs for example

👍